draft-packages/filter-menu-button/docs/FilterMenuButton.stories.tsx

```diff
@@ -242,3 +242,163 @@ export const DefaultWithChildrenAdvancedFilter = () => {
+                  labelText={trait.label}
+                />
+              ))}
+            </CheckboxGroup>
+          </div>
+          <div className={styles.buttons}>
+            <Button
+              secondary={true}
+              fullWidth
+              label="Done"
+              onClick={hideDropdown1}
+            />
+          </div>
+        </>
+      </FilterMenuButton>
+      <FilterMenuButton
+        id={"filter-menu-button--simple2"}
+        labelText={"Animal traits autoHide off"}
+        metadata={checkedTraits}
+        isDropdownVisible={isDropdownVisible2}
+        toggleDropdown={toggleDropdown2}
+        hideDropdown={hideDropdown2}
+        autoHide="off"
+      >
+        <>
+          <div className={styles.content}>
+            <CheckboxGroup labelText="Traits">
+              {DROPDOWN_OPTIONS.map(trait => (
+                <CheckboxField
+                  key={trait.id}
+                  onCheck={() => onCheckboxChange(trait)}
+                  id={`checkbox-${trait.id}`}
+                  checkedStatus={
+                    appliedFilters.find(({ id }) => id === trait.id)
+                      ? "on"
+                      : "off"
+                  }
+                  labelText={trait.label}
+                />
+              ))}
+            </CheckboxGroup>
+          </div>
+          <div className={styles.buttons}>
+            <Button
+              secondary={true}
+              fullWidth
+              label="Done"
+              onClick={hideDropdown2}
+            />
+          </div>
+        </>
+      </FilterMenuButton>
+      <FilterMenuButton
+        id={"filter-menu-button--simple3"}
+        labelText={"Animal traits Outside click only"}
+        metadata={checkedTraits}
+        isDropdownVisible={isDropdownVisible3}
+        toggleDropdown={toggleDropdown3}
+        hideDropdown={hideDropdown3}
+        autoHide="outside-click-only"
+      >
+        <>
+          <div className={styles.content}>
+            <CheckboxGroup labelText="Traits">
+              {DROPDOWN_OPTIONS.map(trait => (
+                <CheckboxField
+                  key={trait.id}
+                  onCheck={() => onCheckboxChange(trait)}
+                  id={`checkbox-${trait.id}`}
+                  checkedStatus={
+                    appliedFilters.find(({ id }) => id === trait.id)
+                      ? "on"
+                      : "off"
+                  }
+                  labelText={trait.label}
+                />
+              ))}
+            </CheckboxGroup>
+          </div>
+          <div className={styles.buttons}>
+            <Button
+              secondary={true}
+              fullWidth
+              label="Done"
+              onClick={hideDropdown3}
+            />
+          </div>
+        </>
+      </FilterMenuButton>
+    </>
+  )
+}
+
+AutoHideBehaviours.storyName = "Auto hide behaviours"
+AutoHideBehaviours.parameters = { chromatic: { disable: false } }
```

Comment:
```suggestion
```

Since autohide is functional and not visually different, we should just not send it to Chromatic.